cinema/serializers.py

```diff
@@ -1 +1,106 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Movie,
+    Genre,
+    Actor,
+    MovieSession,
+    CinemaHall,
+    Order,
+    Ticket
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ("id", "name",)
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
```

Comment:
The `full_name` field is a property in the `Actor` model, not a database field. Including it in the `fields` list of `ActorSerializer` might cause issues during serialization. Consider using a `SerializerMethodField` to handle this property.